flink-ml-core/src/main/java/org/apache/flink/ml/linalg/BLAS.java

```diff
@@ -26,9 +26,57 @@
     private static final dev.ludovic.netlib.BLAS JAVA_BLAS =
             dev.ludovic.netlib.JavaBLAS.getInstance();
 
-    /** y += a * x . */
+    /**
+     * \sum_i |x_i| .
+     *
+     * @param x x
+     * @return \sum_i |x_i|
+     */
+    public static double asum(DenseVector x) {
+        return JAVA_BLAS.dasum(x.size(), x.values, 0, 1);
+    }
+
+    /**
+     * y += a * x .
+     *
+     * @param a a
```

Comment:
This `@param a a` seems to redundant as it does not provide any useful information.

Could we remove these and just keep `y += a * x`? This could be similar to Spark's BLAS Java doc.

Same for other methods' Java doc.